frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx

```diff
@@ -277,8 +277,6 @@ export const SubscriptionsPage: React.SFC<SubscriptionsPageProps> = (props) => {
       ]}
       flatten={(resources) => _.get(resources.subscription, 'data', [])}
       title="Subscriptions"
-      helpText="Operator Subscriptions keep your services up to date by tracking a channel in a package. The approval strategy determines either manual or automatic updates."
```

Comment:
I removed the help text as in #4760. It doesn't work well in Home -> Search where this list appears.